suzieq/engines/pandas/address.py

```diff
@@ -80,14 +81,17 @@ def get(self, **kwargs) -> pd.DataFrame:
         df = df.rename({'master': 'vrf'}, axis=1) \
             .replace({'vrf': {'': 'default'}})
 
-        if 4 in addr_types:
-            df = df.explode('ipAddressList').fillna({'ipAddressList': ''})
+        addrcols = []
+        if 4 in addr_types or ipvers in ["v4", ""]:
+            # df = df.explode('ipAddressList').fillna({'ipAddressList': ''})
+            addrcols.append('ipAddressList')
 
-        if 6 in addr_types:
-            df = df.explode('ip6AddressList').fillna({'ip6AddressList': ''})
+        if 6 in addr_types or ipvers in ["v6", ""]:
+            # df = df.explode('ip6AddressList').fillna({'ip6AddressList': ''})
+            addrcols.append('ip6AddressList')
 
-        if 'ipAddress' in columns or columns == ['*']:
-            ndf = pd.DataFrame(df[['ipAddressList', 'ip6AddressList']].agg(
+        if ('ipAddress' in columns or (columns == ['*'])) and not ipvers:
+            ndf = pd.DataFrame(df[addrcols].agg(
```

Comment:
I think the result is going to be different from the older version since you are no more using the `df.explode` function. This result format modification was planned or not?

Reply:
Yes, that was intended. When I do a show, I didn't expect the addresses to be exploded, but they are now, after the prefix support was added. So, I fixed it.